readme.md

```diff
@@ -50,14 +53,21 @@ Length of the returned string.
 
 ##### type
 
+<<<<<<< HEAD
 Type: `string`\
 Default: `'hex'`\
-Values: `'hex' | 'base64' | 'url-safe' | 'numeric'`
+Values: `'hex' | 'base64' | 'url-safe' | 'numeric'` | `'distinguishable'`
 
 Use only characters from a predefined set of allowed characters.
 
+"distinguishable" set contains only uppercase characters which are not easily confused:
+
+    CDEHKMPRTUWXY012458
+
 Cannot be set at the same time as the `characters` option.
 
+This character set is useful if you need to print out a short string that you'd like user's to read and type back in with minimal errors. For example, reading a code off of a screen that needs to be typed into a phone to connect the two devices.
```

Comment:
This should be together with the other text about `distinguishable`. This should also be in the TS file. They should be in sync.